amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnector.scala

```diff
@@ -66,7 +66,7 @@ private[amqp] trait AmqpConnector {
 private[amqp] trait AmqpConnectorLogic { this: GraphStageLogic =>
 
   private var connection: Connection = _
-  protected var channel: Channel = _
+  protected implicit var channel: Channel = _
```

Comment:
Shouldn't need to be implicit anymore, right?

Reply:
Yup, I'll revert this.